.teamcity/Gradle_Check/model/FunctionalTestBucketProvider.kt

```diff
@@ -220,7 +180,7 @@ class GradleVersionRangeCrossVersionTestBucket(private val startInclusive: Strin
         )
 }
 
-class LargeSubprojectSplitBucket(val subproject: GradleSubproject, val number: Int, val include: Boolean, val classes: List<TestClassTime>) : BuildTypeBucket by subproject {
+class LargeSubprojectSplitBucket(val subproject: GradleSubproject, number: Int, val include: Boolean, val classes: List<TestClassTime>) : BuildTypeBucket by subproject {
```

Comment:
Why this change?

Reply:
Idea tells me that `number` is never used as a `val`, so I removed the `val` making the property only available while constructing the object.